test/cpp/lite_interpreter/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+set(LITE_INTERPRETER_TEST_DIR "${TORCH_ROOT}/test/cpp/lite_interpreter")
+set(LITE_INTERPRETER_TEST_DIR
+  ${TORCH_ROOT}/test/cpp/lite_interpreter/main.cpp
+  ${TORCH_ROOT}/test/cpp/lite_interpreter/test_cores.cpp
+)
+
+add_executable(test_lite_interpreter ${LITE_INTERPRETER_TEST_DIR})
+target_include_directories(test_lite_interpreter PRIVATE ${ATen_CPU_INCLUDE})
+target_link_libraries(test_lite_interpreter PRIVATE torch gtest)
```

Comment:
This "torch" would be the lib of torch_lite? Just to make sure that when this unit test is added and enabled, both the original tests with "torch", like test_jit, and this test with "torch(lite)" can both run for each PR. Would it be better to have different name, "torch_lite", that is always built in CI?

Reply:
Hmm I thought about it. I actually prefer the name libtorch_lite. The main reason that I didn't use torch_lite is 1) it will introduce lots of nasty if-else in CMake files, like the change in this pr in `android/pytorch_android/CMakeLists.txt` and 2) there are some dependency on the name libtorch from the app. 3) We already have caffe2, libtorch, and it's probably a bit confusing to introduce libtorch_lite. So I'm thinking maybe it's better to keep the name as is for now in this change, and rename it if it's preferred later?